<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="aValue">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="aLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bValue">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>b=</string>
    </property>
   </widget>
   <widget class="QLabel" name="cLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>c=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cValue">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exampleLabel">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>30</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ax^2+bx+c=0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Task 1</string>
    </property>
    <widget class="QPushButton" name="calcBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Result:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ResultLable">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Task 2</string>
    </property>
    <widget class="QLineEdit" name="aSideValue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sideaLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Side a=</string>
     </property>
    </widget>
    <widget class="QLabel" name="sidebLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Side b=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bSideValue">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="RadianRdBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Radian</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DegreeRdBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Degree</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="AngleLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Angle =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AngleValue">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearBtnTask2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CalcBtnTask2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="sideaLabel_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Side c=</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultLableTask2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>711</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Task3</string>
    </property>
    <widget class="QPlainTextEdit" name="FrstPlainTxtEd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ScndPlainTxtEd">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="AddBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add &gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ChngBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Change &lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HtmlBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Insert HTML &gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;&lt; Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear &gt;&gt;</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>aValue</zorder>
   <zorder>aLabel</zorder>
   <zorder>bValue</zorder>
   <zorder>bLabel</zorder>
   <zorder>cLabel</zorder>
   <zorder>cValue</zorder>
   <zorder>exampleLabel</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
